### PR TITLE
Update in-repo READMEs

### PR DESCRIPTION
**Pull request by nonparibus**
## 🌟 What is the purpose of this PR?

Updates the `apps`, `apps/hash`, `apps/hash-external-services` and `blocks` READMEs.

### CODE REVIEW

File: apps/hash-external-services/README.md [hunk 1]
```diff
@@ -4,6 +4,17 @@
 
 This directory contains a number of external services used throughout HASH (Ory Kratos, OpenSearch, Postgres, and Temporal).
 
+### LLM prototyping
+
+This also contains an experimental Docker compose file for prototyping LLM-based services using relevant external services such as a vector database.
+You'll be able to execute the following command to start the prototyping external services:
```
Comment (TimDiekmann) [inline apps/hash-external-services/README.md:10]:
```suggestion
This also contains an experimental Docker compose file for prototyping LLM-based services using relevant external services such as a vector database.
You'll be able to execute the following command from the repository root directory to start the prototyping external services:
```

**Review by TimDiekmann — COMMENTED** [review]
Looks good to me. I have suggested a drive-by improvement, but otherwise happy to approve.

**Review by TimDiekmann — DISMISSED** [review]

File: apps/README.md [hunk 2]
```diff
@@ -19,23 +19,42 @@
...
+| Subdirectory                             | Description                                                                                                |
+| ---------------------------------------- | ---------------------------------------------------------------------------------------------------------- |
+| [`engine`](engine)                       | Codebase for the alpha version of [HASH Engine], a versatile agent-based simulation engine written in Rust |
+| [`intellij-plugin`](intellij-plugin)     | Plugin for JetBrains IntelliJ-based IDEs to assist development in common HASH workflows                    |
+| [`hash-agents`](hash-agents)             | An experimental setup for writing Python-based 'agents' that interface with LLMs.                          |
+| [`hash-ai-worker-py`](hash-ai-worker-py) | A Python-based [Temporal](temporal.io) worker, tasked with executing AI-powered workflows.                 |
+| [`hash-ai-worker-ts`](hash-ai-worker-ts) | A TypeScript-based [Temporal](temporal.io) worker, tasked with executing AI-powered workflows.             |
```
Comment (nonparibus) [inline apps/README.md:58]:
```suggestion
| [`hash-agents`](hash-agents)             | An experimental setup for writing Python-based 'agents' that interface with LLMs                           |
| [`hash-ai-worker-py`](hash-ai-worker-py) | A Python-based [Temporal](temporal.io) worker, tasked with executing AI-powered workflows                  |
| [`hash-ai-worker-ts`](hash-ai-worker-ts) | A TypeScript-based [Temporal](temporal.io) worker, tasked with executing AI-powered workflows              |
```

File: apps/README.md [hunk 3]
```diff
@@ -19,23 +19,42 @@
...
+| `hash-graph`             | The query layer over the main datastore of HASH, its strongly-typed graph.                                                                                                   |
+| `hash-external-services` | Defines the running configurations of external (not internally-developed) services that HASH depends on, such as Postgres, Ory Kratos, and Temporal. _(pending refactoring)_ |
+| `hash-realtime`          | Implements a different view over the graph datastore that allows services to subscribe to realtime updates on entities.                                                      |
+| `hash-search-loader`     | Loads the change-stream published by the realtime service into a search index.                                                                                               |
+| `hash-task-executor`     | An experimental service that allows for the execution of pre-defined tasks. (Pending removal in favor of Temporal)                                                           |
```
Comment (nonparibus) [inline apps/README.md:38]:
```suggestion
```

File: apps/hash/README.md [hunk 4]
```diff
@@ -26,15 +26,16 @@ We will be developing HASH into a production-grade application which can be self
...
-- [hash-realtime](../hash-realtime): provides realtime updates on entities to a collection of subscribers
-- [hash-search-loader](../hash-search-loader): loads the change-stream published by the realtime service into a search index
-- [hash-task-executor](../hash-task-executor): supports the triggered execution of scripts _(temporary solution)_
+- [hash-api](../hash-api)
+- [hash-external-services](../hash-external-services)
+- [hash-frontend](../hash-frontend)
+- [hash-graph](../hash-graph)
+- [hash-realtime](../hash-realtime)
+- [hash-search-loader](../hash-search-loader)
+- [hash-task-executor](../hash-task-executor)
```
Comment (nonparibus) [inline apps/hash/README.md:38]:
```suggestion
```

**Review by TimDiekmann — APPROVED** [review]

**Pull request merged**